examples/sdk-examples/src/examples/basic/HeadlineExample.tsx

```diff
@@ -1,52 +1,51 @@
 // (C) 2007-2022 GoodData Corporation
-import React, { Component } from "react";
+import React from "react";
 import { OnLoadingChanged, OnError } from "@gooddata/sdk-ui";
 import { Headline } from "@gooddata/sdk-ui-charts";
 import { modifyMeasure } from "@gooddata/sdk-model";
 import * as Md from "../../md/full";
+import noop from "lodash/noop";
 
 const FranchiseFees = modifyMeasure(Md.$FranchiseFees, (m) => m.format("#,##0").title("Franchise Fees"));
 const FranchiseFeesAdRoyalty = modifyMeasure(Md.$FranchiseFeesAdRoyalty, (m) => m.format("#,##0"));
 
-export class HeadlineExample extends Component {
-    public onLoadingChanged: OnLoadingChanged = (...params) => {
-        // eslint-disable-next-line no-console
-        return console.log("ColumnChartExample onLoadingChanged", ...params);
+export const HeadlineExample: React.FC = () => {
+    const onLoadingChanged: OnLoadingChanged = (_params) => {
+        // handle the callback here
+        return noop;
```

Comment:
The `return noop;` is not needed. I think just
```
const onLoadingChanged: OnLoadingChanged = () => {
         // handle the callback here
}
```

is enough. Similarly for other cases.